.evergreen/compile.sh

```diff
@@ -19,33 +19,43 @@ fi
 
 OS=$(uname -s | tr '[:upper:]' '[:lower:]')
 
-cd build
-cmake "-DCMAKE_BUILD_TYPE=${BUILD_TYPE}" -DMONGOCXX_ENABLE_SLOW_TESTS=ON -DENABLE_UNINSTALL=ON "$@" ..
+if [ -f /proc/cpuinfo ]; then
+    CONCURRENCY=$(grep -c ^processor /proc/cpuinfo)
+elif which sysctl; then
+    CONCURRENCY=$(sysctl -n hw.logicalcpu)
+else
+    echo "$0: can't figure out what value of -j to pass to 'make'" >&2
+    exit 1
+fi
 
 case "$OS" in
-    darwin|linux)
-        if [ -f /proc/cpuinfo ]; then
-            CONCURRENCY=$(grep -c ^processor /proc/cpuinfo)
-        elif which sysctl; then
-            CONCURRENCY=$(sysctl -n hw.logicalcpu)
-        else
-            echo "$0: can't figure out what value of -j to pass to 'make'" >&2
-            exit 1
-        fi
-        make "-j$CONCURRENCY" all VERBOSE=1
-        make install VERBOSE=1
-        make "-j$CONCURRENCY" examples VERBOSE=1
-        if [ "$RUN_DISTCHECK" ]; then
-                make DISTCHECK_BUILD_OPTS="-j$CONCURRENCY" distcheck VERBOSE=1
-        fi
-        ;;
-    cygwin*)
-        MSBuild.exe /p:Configuration=${BUILD_TYPE} /m ALL_BUILD.vcxproj
-        MSBuild.exe /p:Configuration=${BUILD_TYPE} INSTALL.vcxproj
-        MSBuild.exe /p:Configuration=${BUILD_TYPE} /m examples/examples.vcxproj
-        ;;
-    *)
-        echo "$0: unsupported platform '$OS'" >&2
-        exit 1
-        ;;
+   darwin|linux)
+      CMAKE_BUILD_OPTS="-j $CONCURRENCY"
+      CMAKE_EXAMPLES_TARGET=examples
+      if [ "$RUN_DISTCHECK" ]; then
+         _RUN_DISTCHECK=$RUN_DISTCHECK
+      fi
+      ;;
+
+   cygwin*)
+      CMAKE_BUILD_OPTS="/maxcpucount:$CONCURRENCY"
+      CMAKE_EXAMPLES_TARGET=examples/examples
```

Comment:
I didn't know that the target name for the examples was different on Windows. Do you know off-hand if it just the examples target that differs?

Reply:
That is the only target with that peculiarity.